packages/core/src/builtin-tasks/contract.ts

```diff
@@ -198,6 +210,181 @@ subtask('contract:deploy')
     return tx
   })
 
+interface ContractUpgradeArgs {
+  name?: string
+  binPath?: string
+  migrationPath: string
+  signer?: 'ckb-cli' | 'ckb-cli-multisig'
+  feepayer?: string[]
+  deployer?: string[]
+  feeRate?: number
+  export?: string
+  send: boolean
+}
+subtask('contract:upgrade')
+  .addParam('name', 'name of the contract to be deployed', '', paramTypes.string, true)
+  .addParam('bin-path', 'path of contract bin file', '', paramTypes.path, true)
+  .addParam('migration-path', 'path of migration file')
+  .addParam('feepayer', 'address or multisig config of the transaction fee payer', '', paramTypes.string, false, true)
+  .addParam('deployer', 'address or multisig config of the contract deployer', '', paramTypes.string, false, true)
```

Comment:
I can't get the point of `feepayer` and `deployer`, could we directly use the `deployer` cell to provide the tx fee?